python/lsst/ts/standardscripts/maintel/track_target_and_take_image_gencam.py

```diff
@@ -78,18 +83,30 @@ async def assert_feasibility(self):
         await asyncio.gather(
             self.mtcs.assert_all_enabled(),
             self.mtcs.assert_liveliness(),
-            self.gencam.assert_all_enabled(),
-            self.gencam.assert_liveliness(),
+            *(gencam.assert_all_enabled() for gencam in self.gencam),
+            *(gencam.assert_liveliness() for gencam in self.gencam),
         )
 
     async def configure(self, config):
         await super().configure(config)
-        self.gencam = GenericCamera(
-            self.config.camera_index,
-            self.domain,
-            intended_usage=self.gencam_usage,
-            log=self.log,
-        )
+
+        # Replace the top-level exp_times with the exposure times from the
+        # generic cameras. This information is used internally to fill up
+        # metadata. @tribeiro - Do we want to keep this?
+        self.config.exp_times = []
```

Comment:
So, this is part of the interface we expose to the Scheduler for it tho provide metadata for the script. I would recommend not touching it.